index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>My First Page</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p><a href="/about/index.html">Об авторе</a></p>
```

Comment:
Seems the path to another HTML file is wrong. Try to use "./about/index.html"

Reply:
Thanks. The path has changed. Should already be working.